src/vs/workbench/contrib/scm/browser/scm.contribution.ts

```diff
@@ -262,6 +262,13 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
 			maximum: 50,
 			default: 10
 		},
+		'scm.inputMinLines': {
+			type: 'number',
+			markdownDescription: localize('inputMinLines', "Controls the minimum number of lines that the input will auto-grow from."),
+			minimum: 1,
+			maximum: 50,
+			default: 3
```

Comment:
The default value should be `1` as we do not want to change existing behaviour.

Reply:
Fixed.